content/blog/2019/06/2019-06-25-Phase1-Updates-On-Working-Hours-Plugin

```diff
@@ -0,0 +1,119 @@
+---
```

Comment:
This file needs to the `adoc` extension in order to be included in the blog, e.g.`2019-06-25-Phase1-Updates-On-Working-Hours-Plugin.adoc`